lib/site_settings/deprecated_settings.rb

```diff
@@ -35,6 +35,64 @@ module SiteSettings::DeprecatedSettings
     ["min_trust_level_to_allow_ignore", "ignore_allowed_groups", false, "3.3"],
   ]
 
+  OVERRIDE_TL_GROUP_SETTINGS = %w[
```

Comment:
Maybe a nicer way to do this is to change the `override` param to be an enum too; then it could be something like:

```ruby
OVERRIDE = {
  yes: :yes,
  no: :no,
  tl_group: :tl_group,
}
```

Or something.

Reply:
`"min_trust_level_to_allow_ignore", "ignore_allowed_groups", OVERRIDE[:tl_group], "3.3"`